### ISSUE TITLE
Win_LGPO fails on writing Administrative Template for Remote Assistance

### ISSUE THREAD

**Issue by vangourd** [saltstack/salt#46627]
### Description of Issue/Question

When attempting to apply LGPO for Remote Assistance configuration both the module and the state file fail with the following error message. 

[Full Debug Log](https://pastebin.com/yDiSK6Lu)

```
Traceback (most recent call last):
  File "C:\salt\bin\lib\site-packages\salt\cli\caller.py", line 204, in call
    ret['return'] = func(*args, **kwargs)
  File "C:\salt\bin\lib\site-packages\salt\modules\win_lgpo.py", line 5525, in set_
    raise CommandExecutionError(msg)
CommandExecutionError: Error while attempting to write Administrative Template Policy data.  Some changes may not be applied as expected
Error running 'lgpo.set': Error while attempting to write Administrative Template Policy data.  Some changes may not be applied as expected
```

### Setup

Testing this locally on *Windows 7 Pro* and *Windows 10 Pro* machines all 64Bit. 

**Command to replicate issue**
`salt-call.bat lgpo.set computer_policy = " { ' RA_Unsolicit': {'Configure Offer Remote Access': Enabled, 'Permit remote control of this computer': 'Allow
helpers to remotely control the computer', 'Helpers': ['administrators'] } }" -l debug`

Module correctly identifies the Policy object and its requirements as the command won't validate until I find the proper wording but once I do it fails when applying.

## Minion Version Report
```
Salt Version:
           Salt: 2017.7.4

Dependency Versions:
           cffi: 1.10.0
       cherrypy: unknown
       dateutil: 2.6.0
      docker-py: Not Installed
          gitdb: 2.0.3
      gitpython: 2.1.3
          ioflo: Not Installed
         Jinja2: 2.9.6
        libgit2: Not Installed
        libnacl: 1.6.1
       M2Crypto: Not Installed
           Mako: 1.0.6
   msgpack-pure: Not Installed
 msgpack-python: 0.4.8
   mysql-python: Not Installed
      pycparser: 2.17
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:53:40) [MSC v.1500 64 bit (AMD64)]
   python-gnupg: 0.4.0
         PyYAML: 3.12
          PyZMQ: 16.0.2
           RAET: Not Installed
          smmap: 2.0.3
        timelib: 0.2.4
        Tornado: 4.5.1
            ZMQ: 4.1.6

System Versions:
           dist:
         locale: cp1252
        machine: AMD64
        release: 10
         system: Windows
        version: 10 10.0.16299  Multiprocessor Free
```
## Master Version Report

Salt Version:
           Salt: 2017.7.4
 
Dependency Versions:
           cffi: Not Installed
       cherrypy: 3.2.3
       dateutil: 2.4.2
      docker-py: Not Installed
          gitdb: 0.6.4
      gitpython: 1.0.1
          ioflo: Not Installed
         Jinja2: 2.8
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: 1.0.3
   msgpack-pure: Not Installed
 msgpack-python: 0.5.5
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 2.7.12 (default, Dec  4 2017, 14:50:18)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 15.2.0
           RAET: Not Installed
          smmap: 0.9.0
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.1.4
 
System Versions:
           dist: Ubuntu 16.04 xenial
         locale: UTF-8
        machine: x86_64
        release: 4.4.0-112-generic
         system: Linux
        version: Ubuntu 16.04 xenial

**Comment by Ch3LL** [comment]
ping @twangboy i know you have done some work here recently. any ideas here or maybe its a  known issue?

**Comment by rallytime** [comment]
Closed via #46800
